src/tests/class_tests/openms/data/MetaboliteFeatureDeconvolution_easy_output.consensusXML

```diff
@@ -14,13 +14,6 @@
 				<element map="0" id="121" rt="0" mz="912.31" it="22004" charge="1"/>
 				<element map="0" id="122" rt="0" mz="456.655" it="22004" charge="2"/>
 			</groupedElementList>
-			<UserParam type="int" name="map_idx" value="0"/>
-			<UserParam type="string" name="dc_charge_adducts" value="H2"/>
-			<UserParam type="stringList" name="adducts" value="[[M+2H]2+]"/>
-			<UserParam type="float" name="dc_charge_adduct_mass" value="2.0156500638"/>
-			<UserParam type="int" name="is_backbone" value="1"/>
-			<UserParam type="string" name="Local" value="2:1"/>
-			<UserParam type="string" name="CP" value="2(H2):1(H1) Score: 0.699999982319689"/>
```

Comment:
Those are scores of one specific adduct pair (i.e. edge), and should not be part of the Consensus Feature. They're still in the pairs output.

They were leftover artifacts in the Consensus Feature because those derive from the pairs (in short a Consensus Feature  is constructed by merging agreeing pairs, with one pair as seed).